projects/observability/src/shared/dashboard/widgets/charts/cartesian-widget/cartesian-widget-renderer.component.ts

```diff
@@ -72,15 +74,29 @@ export class CartesianWidgetRendererComponent<TData> extends InteractiveDataWidg
     );
   }
 
-  protected buildDataObservable(): Observable<Series<TData>[]> {
-    return this.fetcher ? this.buildSeries(this.fetcher, this.selectedInterval) : NEVER;
+  protected buildDataObservable(): Observable<CartesianData<TSeriesInterval>> {
+    if (!this.fetcher) {
+      return of({ series: [], bands: [] });
```

Comment:
if fetcher is not yet defined we should return never rather than empty (we want to show loading while waiting for the fetcher, not empty data state)